DeltaCore/UI/Controller/ControllerView.swift

```diff
@@ -79,6 +79,7 @@ public class ControllerView: UIView, GameController
     public var overrideControllerSkinSize: ControllerSkin.Size?
     
     public var translucentControllerSkinOpacity: CGFloat = 0.7
+    public var isControllerSkinDebugModeEnabled: Bool = false
```

Comment:
Since we're moving away from controller skin-based debug mode, isDebugModeEnabled would be a more accurate name IMO

```suggestion
    public var isDebugModeEnabled: Bool = false
```